cmd/kube-apiserver/app/server.go

```diff
@@ -299,7 +299,18 @@ func Run(s *options.ServerRunOptions) error {
 	}
 
 	admissionControlPluginNames := strings.Split(s.GenericServerRunOptions.AdmissionControl, ",")
-	pluginInitializer := kubeadmission.NewPluginInitializer(client, sharedInformers, apiAuthorizer)
+	var cloudConfigReader *os.File
+
+	if s.CloudProvider.CloudConfigFile != "" {
+		cloudConfigReader, err = os.Open(s.CloudProvider.CloudConfigFile)
+		if err != nil {
+			glog.Errorf("Error opening cloud configuration file %s: %#v", s.CloudProvider.CloudConfigFile, err)
```

Comment:
Because I intended this change to be backward compatible in the sense - a misconfigured apiserver shouldn't start crashing after this change.

It could be argued that, may be it should crash if cloud provider configuration is indeed broken but I thought it is best to least obstrusive.